Documentation/building/linux-instructions.md

```diff
@@ -66,6 +66,11 @@ ellismg@linux:~$ sudo apt-get update
 ellismg@linux:~$ sudo apt-get install mono-devel
 ```
 
+Note that for Debian, you will need libgdiplus, which requires a second package repository:
```

Comment:
libgdiplus is actually installed on other systems too because it's a transitive dependency of mono-devel (somewhere down the chain something depends on System.Drawing...), but it depends on libjpeg8 and Debian 8.0+ moved to libjpeg62-turbo so we need this shim repo.